crates/modelardb_server/src/storage/compressed_data_manager.rs

```diff
@@ -269,13 +271,16 @@ impl CompressedDataManager {
         Ok(table_relevant_apache_parquet_files)
     }
 
-    /// Save [`CompressedDataBuffers`](CompressedDataBuffer) to disk until the reserved memory limit
-    /// is no longer exceeded. If all of the data is saved successfully, return [`Ok`], otherwise
-    /// return [`IOError`].
-    async fn save_compressed_data_to_free_memory(&mut self) -> Result<(), IOError> {
+    /// Save [`CompressedDataBuffers`](CompressedDataBuffer) to disk until at least `size_in_bytes`
+    /// bytes of memory is available . If all of the data is saved successfully, return [`Ok`],
```

Comment:
Very very minor but it seems there is an extra space after `available`.

Reply:
Fixed.